JavaScript/Topics/16.Cookies/README.md

```diff
@@ -0,0 +1,223 @@
+# What are Cookies?
+A **cookie** is an amount of information that persists between a server-side and a client-side. A web browser stores this information at the time of browsing.
+
+A cookie contains the information as a string generally in the form of a name-value pair separated by semi-colons. It maintains the state of a user and remembers the user's information among all the web pages.
+
+Cookies are saved in name-value pairs like:
+```js
+username = John Doe
+```
+# How Cookies Works?
+
+- When a user sends a request to the server, then each of that request is treated as a new request sent by the different user.
+- So, to recognize the old user, we need to add the cookie with the response from the server.
+- **Browser**, at the client-side.
+- Now, whenever a user sends a request to the server, the cookie is added with that request automatically. Due to the cookie, the server recognizes the users.
+
+<p align="center">
+<img src="cookie.png" height=40% width=40% align="middle">
+</p>
+
+# Create a Cookie
+With JavaScript, a cookie can be created like this:
+```js
+document.cookie = "username=John Doe; expires=Thu, 18 Dec 2013 12:00:00 UTC; path=/";
+```
+
```

Comment:
Show the cookie is stored in browser with a photo of the dev tools

Reply:
> Show the cookie is stored in browser with a photo of the dev tools

but if I add the image file I have to create a new PR ig